### PR TITLE
Bug 1688846: Point contributing docs from Bugzilla to GitHub

### PR DESCRIPTION
**Pull request by mathjazz**
One of the last steps of the migration of Pontoon bugs from Bugzilla to GitHub is to update the docs, which means merging this PR. So let's use this PR to track the process and collect feedback.

## Before Migration

- [x] Coordinate [retirement of Webtools::Pontoon](https://bugzilla.mozilla.org/show_bug.cgi?id=1728575) with BMO team.
- [x] Update [bugzilla2github](https://github.com/mathjazz/bugzilla2github/tree/pontoon-migration) script to suit our needs.
- [x] Perform test bug export/issue import into a [test repository](https://github.com/mathjazz/pontoon/issues) using the above script.
- [x] Figure out and document the plan to [report security vulnerabilities](https://bugzilla.mozilla.org/show_bug.cgi?id=1730480) ([security policy](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository), [existing security bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1261893)).
- [x] Announce when we'll be making migration on Pontoon and L10n Matrix.

## Migration

- [x] Let BMO [retire Webtools::Pontoon](https://bugzilla.mozilla.org/show_bug.cgi?id=1728575).
- [x] Enable Issues on https://github.com/mozilla/pontoon/.
- [x] Export [existing bugs](https://bugzilla.mozilla.org/buglist.cgi?product=Webtools&component=Pontoon&resolution=---&list_id=15669458) from Webtools::Pontoon and import them into https://github.com/mozilla/pontoon/.
- [x] Let BMO [close open bugs in Webtools::Pontoon](https://bugzilla.mozilla.org/show_bug.cgi?id=1728575).

## After Migration

- [x] Update docs by merging this PR.
- [ ] Announce migration (Mozilla L10n Blog, Mozilla L10n Twitter, Pontoon/L10n Matrix, Pontoon/L10n Discourse).
- [ ] Replace all `bug 123456` comments in the code with the respective issues.

### CODE REVIEW

File: docs/dev/first-contribution.rst [hunk 1]
```diff
@@ -112,20 +112,14 @@ have a list of mentored bugs that are appropriate to work on first, and
...
-We maintain a list of what we deem "good first bugs". They are marked as
-mentored on bugzilla, and can be found quickly on our `Pontoon wiki
-page <https://wiki.mozilla.org/L10n:Pontoon#Get_involved>`_ or through
-`this bugzilla
-search <https://bugzilla.mozilla.org/buglist.cgi?f1=bug_mentor&list_id=15050149&o1=isnotempty&resolution=---&classification=Server%20Software&query_format=advanced&emailbug_mentor1=1&component=Pontoon&product=Webtools>`_.
-Look through that list for unassigned bugs (marked as ``NEW``), choose
+`GitHub <https://github.com/mozilla/pontoon/issues>`_, where we maintain
+a list of what we deem `good first bugs <https://github.com/mozilla/pontoon/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Bug%22>`_.
+
+Look through that list for unassigned bugs, choose
```
Comment (Pike) [inline docs/dev/first-contribution.rst:118]:
```suggestion
Look through that list for unassigned issues, choose
```

**Comment by mathjazz** [comment]
I've made a test import into https://github.com/mathjazz/pontoon/issues and I'd be happy to hear comments.

The script imports:
- bug title
- bug description and user story (joined into the first issue comment)
- bug creation date
- bug reporter ([maps BZ to GH accounts for most common users](https://github.com/mathjazz/bugzilla2github/blob/pontoon-migration/bugzilla2github#L84))
- assignee ([maps BZ to GH accounts for most common users](https://github.com/mathjazz/bugzilla2github/blob/pontoon-migration/bugzilla2github#L84))
- CCs ([maps BZ to GH accounts for most common users](https://github.com/mathjazz/bugzilla2github/blob/pontoon-migration/bugzilla2github#L84))
- blockers
- dependencies
- see also
- bug priority (as P1-P5 labels)
- bug type (as bug, enhancement, task labels)
- mentored bug status (as Good First Bug label)
- comments (incl. creation date and author)

The import script doesn't trigger notifications.

Assignees are not set in the test repo, because they would need permission to be set.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -10,14 +10,12 @@ Pontoon source code is available via GitHub:
...
 
-Bugs
-====
-
-All bugs are tracked in `<https://bugzilla.mozilla.org/>`_.
+Issues
+======
 
-Write up a new bug:
+All bugs and ideas are tracked in `GitHub <https://github.com/mozilla/pontoon/issues>`_.
```
Comment (Pike) [inline CONTRIBUTING.rst:16]: You've been pretty consistent in nomenclature towards issues, this one still says bug. Nit.

```suggestion
All issues and ideas are tracked in `GitHub <https://github.com/mozilla/pontoon/issues>`_.
```

File: README.md [hunk 3]
```diff
@@ -40,8 +40,8 @@ database, run tests, and send your contribution.
 If you want to go further, you can:
 
 * Check out development roadmap on the [wiki](https://wiki.mozilla.org/Pontoon)
```
Comment (Pike) [inline README.md:42]: Moving the roadmap off the wiki is a follow-up?

File: contribute.json [hunk 4]
```diff
@@ -17,9 +17,9 @@
         "mailing-list": "https://discourse.mozilla.org/c/pontoon"
     },
```
Comment (Pike) [inline contribute.json:18]: https://wiki.mozilla.org/Webdev/GetInvolved/pontoon.mozilla.org wants an edit post-migration, too.

**Review by Pike — CHANGES_REQUESTED** [review]
A few things are still missing. Not blocking, but I mark them as request-changes to see them at least in follow-ups.

`specs/template` mentions bugzilla.

https://github.com/mozilla/pontoon/pull/1999/files#diff-76e7ca2f2843ac5f659b45382768c1b64bf8704e8d3c6189c4e3623f5d465bc2R163 and below is still using bugzilla-style commit messages.

The list of post-migration TODOs should be amended by replacing all `bug 123456` comments in the code with the respective issues, in particular as the bugs don't link to issues.

File: README.md (hunk 3 above)
Comment (mathjazz) [reply README.md:42]: Yes.

**Review by Pike — APPROVED** [review]
LGTM

**Pull request merged**